tableau_utilities/scripts/datasource.py

```diff
@@ -11,6 +11,27 @@
 from tableau_utilities.tableau_server.tableau_server import TableauServer
 
 
+def create_column(name: str, persona: dict):
+    """ Creates the tfo column object with the minimum required fields to add a column
+
+    Args:
+        name: The name for the column. This must be enclosed in square brackets and will become the local-name in code
```

Comment:
```suggestion
        name: The name for the Column, and local_name of the Metadata Record.
```
The `Column` object will account for brackets. So the user can pass `column_name` or `[column_name]`, and they will be treated the same.